docs/user/tutorials/01-30-expose-tcp-gateway-api.md

```diff
@@ -0,0 +1,129 @@
+# Use Gateway API to Expose HTTPBin
+
+This tutorial shows how to expose an HTTPBin Service using Gateway API.
+
+> [!WARNING]
+> This tutorial is based on experimental Istio module.
+> Exposing a workload to the outside world is a potential security vulnerability, so tread carefully. This example is not meant to be used in production environment. 
+
+## Prerequisites
+
+* Experimental Istio module installation
+
+## Steps
+
+### Configure Gateway API alpha support
+
+Edit Istio CR by setting `enableAlphaGatewayAPI` to true. Your Istio CR should look:
+
+    ```
```

Comment:
We could simplify that for the user by providing the kubectl patch commands, e.g.
```bash
kubectl patch istios/default -n kyma-system --type merge -p '{"spec":{"experimental":{"pilot": {"enableAlphaGatewayAPI": true}}}}'
```